hadoop-ozone/dist/src/main/compose/ozone-topology/test.sh

```diff
@@ -15,6 +15,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DATANODE_COUNT=4
```

Comment:
shellcheck:1: warning: DATANODE_COUNT appears unused. Verify it or export it. [SC2034]